Orange/data/table.py

```diff
@@ -1280,10 +1282,13 @@ def _compute_contingency(self, col_vars=None, row_var=None):
             raise ValueError("contingency can be computed only for discrete "
                              "and continuous values")
 
-        if any(var.is_continuous for var in col_desc):
-            if bn.countnans(row_data):
-                raise ValueError("cannot compute contigencies with missing "
-                                 "row data")
+        unknown_rows = bn.countnans(row_data)
+        if unknown_rows:
+            nan_inds = np.isnan(row_data)
```

Comment:
`np.isnan` will not work if `row_data` is a column from metas that also contained a StringVariable because its dtype will be np.object (isnan needs floats)